definition_validator/validator.cc

```diff
@@ -249,8 +276,9 @@ void validateCompatibleOverride(const core::Context ctx, core::MethodRef superMe
     if (auto leftRest = left.pos.rest) {
         if (!right.pos.rest) {
             if (auto e = ctx.beginError(methodDef.declLoc, core::errors::Resolver::BadMethodOverride)) {
-                e.setHeader("{} method `{}` must accept *`{}`", implementationOf(ctx, superMethod),
-                            superMethod.show(ctx), leftRest->get().show(ctx));
+                auto splatParts = formatSplat(&leftRest->get(), SplatKind::ARG, ctx);
```

Comment:
In C++ there's very few cases where we want to take a pointer to something. Most of the time, you want to use a reference:


```cpp
formatSplat(const core::ArgInfo &arg, ...) {

// ...

formatSplat(leftRest->get(), ...);
```